### ISSUE TITLE
Windows version not updated

### ISSUE THREAD

**Issue by Gheerdan** [ericfitzgerald/KingmakerCharacterEditor#23]
So, I am a novice programmer at best, I barely understand what this website supports, I am just trying to get this character/save game editor to work. All of the updates that you have posted, don't seem to be enacted. I have redownloaded, but its still the original program. I tried opening the files that you have modified, and the new code seems to be there. 

I download the .zip

unpack it with 7zip

run the executable. 

Am I missing something?

**Comment by MischaZe** [comment]
Yes, you have to compile it yourself.

Download the newest code via clone/download zip. Unzip.

Download nodejs from https://nodejs.org/en/ and install it.

Run nodejs command prompt as administrator (!). Don't use the node.exe in the program folder. Instead, type cmd in your start menu/start screen (depending on win version), then it should pop up. right click, run as admin. 

There should be a console active. Use "cd.." and "cd" to where you unzipped the new code. Unless you changed the name, the folder should be called kingmakercharactereditor-master. Cd into that. Cd into the subfolder with the identical name. (Or skip that step by taking the files our of the subfolder into the main folder. There isn't anything else inside it). You can always type "dir" to see whats in the currently active folder.

Now, type "npm install". If you did everything right, this should start a process of extracting quite a bit of things and also some downloads from github, so you need internet connection. If you get an early failure notie, you did something wrong. Probably you didn't run as admin (everything connected to "access/process denied") or you are in the wrong folder ("package.json there is no such file etc.").

Even with a correct install you will probably get some warnings, which shouldn't matter. However, if you are unsure, check them via audit/auditfix.

When the installation is finished, type npm run build to start the editor. 

Whenever you want to run it again afterwards, you don't have to install again, but you still have to access it via console, manouvering to the right location, type npm run build. I'm quite a noob myself, so I decided that typing these few lines is less effort than trying to set up an electron app via electron builder.

If anyone has an easy tutorial for doing so, I would really appreciate it :)

Also there might be more elegant ways to compile the stuff. As I said, I'm not realy experienced. All I can tell you is that the method above worked for me and I'm now running around with a Pally Valerie and a Monk Renognar.

**Comment by Gheerdan** [comment]
MischZe,

That was awesome. I got it to do what you said. Thank you so much. 

One issue, it ummm... ran the not updated version I was already using. 

So, what exact file or files should I download and in what order? 

I had the kingmakercharactereditor-win32-x64.zip which was the old one, and I tried the npm thing with that, it didn't work

I downloaded the source.zip, that is the one it was able to unpack. When it unpacks it becomes KingmakerCharacterEditor-1.0.3

Do I need both? Should I put one inside the other? 

I am almost there, but I am definitely still missing something.
